### PR TITLE
Version 1.8.0

### PR DESCRIPTION
**Pull request by nitriques**
This version brings two new features:

- Add the Member Extension instance to delegates 49cf3d5
- Add the MembersPreLogin delegate 738c9d2

Since the release is included, to not merge via github (I'll fast forward it).

### CODE REVIEW

**Comment by nilshoerrmann** [comment]
I won‘t be able to review this anytime soon.

**Comment by jensscherbl** [comment]
> I won‘t be able to review this anytime soon.

Me neither, sorry. As previously mentioned, I’m not using the members extension in any project at the moment and haven’t been involved for too long.

File: extension.driver.php [hunk 1]
```diff
@@ -928,7 +933,28 @@ public function checkFrontendPagePermissions($context) {
...
+				 *  '/frontend/'
+				 * @param boolean can-logged-in
+				 *  If the current login is valid or not
+				 * @param extensionMember $driver
+				 *  The Member Extension driver
+				 * @param array $errors
+				 *  The error array
+				 */
+				Symphony::ExtensionManager()->notifyMembers('MembersPreLogin', '/frontend/', array(
+					'can-logged-in' => &$canLogIn,
```
Comment (brendo) [inline extension.driver.php]: should this be named just `can-log-in`? `can-logged-in` a little awkward to native english speaker.

Comment (brendo) [reply extension.driver.php]: Er actually, it should `can_log_in` to be consistent with how other variables are named in this extension's delegates.

File: extension.driver.php [hunk 2]
```diff
@@ -941,15 +967,28 @@ public function checkFrontendPagePermissions($context) {
...
+					 * @param extensionMember $driver
+					 *  The Member Extension driver
+					 * @param array $errors
+					 *  The error array
 					 */
 					Symphony::ExtensionManager()->notifyMembers('MembersPostLogin', '/frontend/', array(
 						'member_id' => $this->getMemberDriver()->getMemberID(),
-						'member' => $this->getMemberDriver()->getMember()
+						'member' => $this->getMemberDriver()->getMember(),
+						'is-logged-in' => &$isLoggedIn,
```
Comment (brendo) [inline extension.driver.php]: As above, `is_logged_in`?

File: extension.driver.php (hunk 1 above)
Comment (nitriques) [reply extension.driver.php]: 👍

Comment (nitriques) [reply extension.driver.php]: Well, I did found why I made it like that: There is `is-logged-in` (you see my copy/paste do you?) So I've made it 'can-log-in'

Comment (nitriques) [reply extension.driver.php]: If it is ok, you can just approve the PR and I'll release it. Thanks.

**Pull request merged**